### DISCUSSION TITLE
New EcuBus-Pro Done！

### DISCUSSION THREAD

**Discussion by frankie-zeng** [ecubus/EcuBus-Pro#55] [Show and tell]
Upgrade old EcuBus to EcuBus-Pro. Welcome to use it.

**Comment by LeventCelik** [comment]
Unexpected good news!

Does it still work with NXP Unified Bootloader project?

**Reply by frankie-zeng** [reply]
sure, see here: https://github.com/ecubus/EcuBus-Pro/tree/master/resources/examples/nxp_bootloader

**Reply by LeventCelik** [reply]
Thanks. I'm currently using S32K144-Q100 and S32K3X4-T172. I'll try with both boards.
